internal/commands/untrust_builder.go

```diff
@@ -28,6 +29,12 @@ func UntrustBuilder(logger logging.Logger, cfg config.Config) *cobra.Command {
 			}
 
 			if len(existingBuilders) == len(cfg.TrustedBuilders) {
```

Comment:
I would add a comment - "builderName isn't part of the trusted builders list" or replace this line with a simple function with a meaningful name.